apps/tax-tribunals/base/kustomize.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: tax-tribunals
+  namespace: flux-system
+spec:
+  path: ./apps/tax-tribunals/${ENVIRONMENT}/base
+  postBuild:
+    substitute:
+      NAMESPACE: "tax-tribunals"
+      TEAM_NOTIFICATION_CHANNEL: "pet"
+      TEAM_AAD_GROUP_ID: "dcd_group_pet_v2"
```

Comment:
Is it going to use pet AAD group?

Reply:
It was discussed, yes, but better safe than sorry. Let me confirm.

Reply:
Yes confirmed